generators/base-simple-application/types.d.ts

```diff
@@ -0,0 +1,48 @@
+import type { PackageJson } from 'type-fest';
+import type CoreGenerator from '../base-core/generator.ts';
+import { type Config as BaseConfig, type Features as BaseFeatures, type Options as BaseOptions } from '../base/index.js';
+
+export type Application = {
```

Comment:
Should be called BaseSimpleApplication

Reply:
I'am trying to follow Application/Entity/Config/Options on types.d.ts for every generator.